keps/NNNN-kep-template/README.md

```diff
@@ -473,6 +473,21 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+###### How can an end user leveraging this feature know that it is working?
```

Comment:
nit: I'm assuming that it's about every "instance of the feature" (however we call it), right?
E.g. if this is a pod-related feature, I would like to answer that for every pod separately?

I like the question, but I think we may try to be a bit more precise on it - wdyt?